config.json

```diff
@@ -184,6 +184,13 @@
       ]
     },
     {
+      "slug": "poker",
+      "difficulty": 4,
```

Comment:
So while the concepts involved aren't super difficult, there's a _lot_ of logic involved here, which makes me think that it should be a little higher up the difficulty chain. I mean, `list-ops` is pretty basic stuff and that's also a 4. This strikes me initially as around the same level of difficulty as bowling, but not as difficult because there isn't the state management aspect.